extensions/delivery-tracker/src/package.ts

```diff
@@ -1,24 +1,9 @@
 import { Color, Icon } from "@raycast/api";
-import { Delivery } from "./delivery";
+import { Delivery } from "./types/delivery";
+import { Package, PackageMap } from "./types/package";
 
-interface Activity {
-  time: Date;
-  description: string;
-  location: string;
-}
-
-export interface Package {
-  deliveryDate?: Date;
-  delivered: boolean;
-  activity: Activity[];
-}
-
-export interface PackageMap {
-  [key: string]: {
-    packages: Package[];
-    lastUpdated?: Date;
-  };
-}
+// Re-export types for backward compatibility
+export type { Package, PackageMap, Activity } from "./types/package";
```

Comment:
Let's remove these lines and convert any previous users to use the new location.